templates/repo/diff/image_diff.tmpl

```diff
@@ -2,7 +2,7 @@
 <tr>
 	<td colspan="2">
 		<div class="image-diff" data-path-before="{{.root.BeforeRawPath}}/{{PathEscapeSegments .file.OldName}}" data-path-after="{{.root.RawPath}}/{{PathEscapeSegments .file.Name}}">
-			<div class="ui secondary pointing tabular top attached borderless menu new-menu">
+			<div class="ui secondary pointing tabular top attached borderless menu new-menu gt-mb-0">
```

Comment:
Adding `gt-mb-0` also doesn't seem right. It breaks other tabs.

I would always prefer avoid these "fine-tuning" helpers as much as possible, just use a well-designed framework mechanism.


![image](https://github.com/go-gitea/gitea/assets/2114189/970771cf-3cf7-43ef-a6e9-f27b1a8eb087)

Reply:
I will propose a separate fix for the "image-diff", I guess some other code also needs to be rewritten.

Reply:
-> Improve Image Diff UI #26696

Reply:
Without the `gt-mb-0`, the spinner had unnecessary margin on top, but I guess you fixes that in https://github.com/go-gitea/gitea/pull/26696.

Reply:
Merged with main and conflicts have been resolved.